### PR TITLE
bump docker-stacks tag to e7367

### PR DESCRIPTION
**Pull request by minrk**
Get scipy stack back in, which includes sympy.
The previous build inherited from all-spark before pyspark was re-based on scipy.

- [x] waiting for test build

closes jupyter/try.jupyter.org#13

### CODE REVIEW

**Comment by minrk** [comment]
Had to pin activemodel for ruby since we have ruby 2.1, which activemodel 5.0 doesn't support, but it's all working now.

File: Dockerfile [hunk 1]
```diff
@@ -140,8 +141,8 @@ RUN find /home/jovyan/work -name '*.ipynb' -exec jupyter nbconvert --to notebook
 # can reuse the very expensive build of all the above with their own site 
 # customization.
 
-# Expose our custom setup to the installed ipython (for mounting by nginx)
-COPY resources/custom.js /opt/conda/lib/python3.4/site-packages/notebook/static/custom/
+# Install our custom.js
+COPY resources/custom.js /home/jovyan/.jupyter/custom/
```
Comment (parente) [inline Dockerfile]: Better to use $NB_USER.

Comment (minrk) [inline Dockerfile]: Good call. Updated this and various other instances.

**Comment by parente** [comment]
LGTM if the build completed for you.

**Comment by minrk** [comment]
Build succeeded. Pushing...

**Pull request merged**